client/my-sites/marketing/tools/feature-buttons-with-plan-gate.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React, { FunctionComponent } from 'react';
+import page from 'page';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import Button from 'components/button';
+import { getPlan } from 'lib/plans';
+import { hasFeature } from 'state/sites/plans/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+interface MarketingToolsFeatureButtonsWithPlanGateConnectProps {
+	hasPlanFeature: boolean;
+	planTitle: string;
+	selectedSiteSlug: string | null;
+}
+
+interface MarketingToolsFeatureButtonsWithPlanGateOwnProps {
+	buttonText: string;
+	feature: string;
+	handleButtonClick: () => void;
+	minPlanSlug: string;
+}
+
+const MarketingToolsFeatureButtonsWithPlanGate: FunctionComponent<
```

Comment:
Since this component only displays one button, we should probably remove the `s` 😉.

Reply:
I was thinking there are two types of buttons it can produce, but you are right, button is better 👍